SurveyTests/Presentation/Modules/Login/LoginViewModelSpec.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  LoginViewModelSpec.swift
+//  SurveyTests
+//
+//  Created by Khanh on 07/08/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+//  swiftlint:disable function_body_length closure_body_length
+
+import Combine
+import Nimble
+import Quick
+import SwiftUI
+
+@testable import Survey
+
+final class LoginViewModelSpec: QuickSpec {
+
+    var logInUseCase: LoginUseCaseMock!
+    var storeTokenUseCase: StoreTokenUseCaseMock!
+    var input: LoginViewModel.Input!
+    var output: LoginViewModel.Output!
+
+    override func spec() {
+        var viewModel: LoginViewModel!
+        let logInTrigger = PassthroughSubject<Void, Never>()
+
+        describe("a LoginViewModel") {
+
+            beforeEach {
+                self.logInUseCase = LoginUseCaseMock()
+                self.storeTokenUseCase = StoreTokenUseCaseMock()
+                viewModel = LoginViewModel(loginUseCase: self.logInUseCase, storeUseCase: self.storeTokenUseCase)
+
+                self.input = LoginViewModel.Input(logInTrigger: logInTrigger.eraseToAnyPublisher())
+                self.output = viewModel.transform(self.input)
+            }
+
+            afterEach {
+                viewModel = nil
+            }
+
+            context("when login with email and password returns success") {
+
+                it("returns output isLoggedInSuccessfully with true") {
+                    self.input.email = "nkhanh44@nimblehq.co"
```

Comment:
Please don't use real email/password. Use `example@mail.com` instead.

Reply:
Thanks, I will fix this 🙇. In case of UITest where I have to use real email/password to run the flow, could i use real email/password? 🙏

Reply:
Normally we use a mock API (Postman Mock API) and switch the base url with env. This IC don't need to do that so you can use real email/password. Just make sure not to public the repo (your account) 🤣

Reply:
Thanks, that's nice information 👍 

> Just make sure not to public the repo (your account)

🥶 🥶 I will notice this.